src/posts/podcast-ep04-a-sneak-peek-into-upcloud-kubernetes-service.md

```diff
@@ -0,0 +1,432 @@
+---
+type: Blog
+title: A sneak peek into UpCloud Kubernetes Service
+subheading: In this episode of The Continuous & Cloud Podcast, Jacob and Lauri chat with Ville Törhönen, Product Owner at UpCloud, about UpCloud’s upcoming managed Kubernetes Service (UKS).
+authors:
+- jlarfors
+- lsuomalainen
+tags:
+- Kubernetes
+- Cloud
```

Comment:
Do we want to add an UpCloud tag as well? If we will do more things with UpCloud it might be worth doing it... But we should also try to limit the total number of tags used. So not sure.

Reply:
Yes, I guess we have HashiCorp as a tag. So just following the same convention.